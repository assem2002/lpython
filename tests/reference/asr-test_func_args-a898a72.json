{
    "basename": "asr-test_func_args-a898a72",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_func_args.py",
    "infile_hash": "5fae2ad01849e5ba3fca0a85455a92199a779be76b908fed30fb6b7d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_func_args-a898a72.stderr",
    "stderr_hash": "81c7b0c6d133c456c6d08779ade1f7c3a4f0eb32bb935caa2ff0286e",
    "returncode": 2
}