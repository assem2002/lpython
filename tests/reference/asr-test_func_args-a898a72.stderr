semantic error: Number of arguments does not match in the function call
  --> tests/errors/test_func_args.py:11:11
   |
11 |     ans = fib(35, 10)
   |           ^^^^^^^^^^^ (found: '2', expected: '1')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
